tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -436,13 +436,22 @@ stages:
           script: |
             mkdir -p $HOME/.onnx
             docker run --rm \
+              --device=/dev/kfd \
+              --device=/dev/dri \
+              --group-add $(video) \
+              --group-add $(render) \
+              --privileged \
+              --ipc=host \
+              --network=host \
+              --cap-add=SYS_PTRACE \
```

Comment:
You already have line 443. Do you still need line 446?

Reply:
I am not familiar about ROCM image. I just copied and pasted from previous calls to docker run in this file